### PR TITLE
Reportback change photo button

### PR DESCRIPTION
**Pull request by sbsmith86**
Refreshes the page when a user clicks "change photo"

@weerd @DFurnes

### CODE REVIEW

**Comment by weerd** [comment]
For a temp fix, looks good! :+1:

File: lib/themes/dosomething/paraneue_dosomething/js/images/ImageUploadBeta.js [hunk 1]
```diff
@@ -106,6 +106,13 @@ define(function(require) {
       event.preventDefault();
       Modal.close(false);
     });
+
+    // Close the modal when a user wants to change their photo.
+    this.$cropModal.find(".-change").click(function(event) {
+      event.preventDefault();
+      window.location.href = "#prove";
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/js/images/ImageUploadBeta.js:113]: You can close an open modal like this! :smile: 

``` js
var Modal = require("neue/modal");
Modal.close();
```

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/js/images/ImageUploadBeta.js:113]: Problem is we're having some issues needing to clear out the file-input file, so we're temporarily doing a page refresh to reset until we can do a proper fix.

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/js/images/ImageUploadBeta.js:113]: Ohhh, tricky.

**Pull request merged**